### DISCUSSION TITLE
High proportion of unspliced counts

### DISCUSSION THREAD

**Discussion by dsg03** [theislab/scvelo#463] [Q&A]
Hello,

Thank you for making this great package!

I have a question about the proportion of unspliced counts. The tutorial states that 10-25% of counts should be unspliced. In addition, the tutorial extracted 2000 highly variable genes after filtering, when the minimum number of shared counts is set at 20. For our data, the proportion of unspliced counts is about 60% - very high. In addition, there are only 23 highly variable genes after the filtering - very low. The cells for our data are Tregs.

Do you have any suggestions?

**Comment by WeilerP** [comment]
23 HVG should be too few to infer dynamics. You should always take a look at your phase portraits to see if they contain information or not. Genes are informative for inference when they exhibit the characteristic almond/football shape.

**Reply by denvercal1234GitHub** [reply]
@dsg03 -- May I ask which tissue is your data coming from? From our data, we also observed very high % of unspliced (10X) from human tonsils. @WeilerP - do you know if high % of unspliced counts would prevent us from having that characteristic almond phase portrait? And, so we should do Cellrank instead? Thanks for your insights.
